### DISCUSSION TITLE
add notification list with workflow

### DISCUSSION THREAD

**Discussion by Giorgos-Hellene** [YetiForceCompany/YetiForceCRM#14301] [Feature improvement]
When i trigger a notification with any workflow, this isn't seems everywhere after completed.

It will be very important if any triggered notification with wokflow to be seem to its related module. 
If i trigger it on Sales orders module it must be seen on Notifications of Sales orders.

I would be very happy if you added this function

**Comment by mariuszkrzaczkowski** [comment]
give a detailed description / show how to trigger an error, your description says nothing

**Reply by Giorgos-Hellene** [reply]
@mariuszkrzaczkowski Hi ! Sorry for my unclear describe.
I have made a video for better results
https://www.loom.com/share/bad1e6e2730a45e7b6ea4baa6ee70258

Αccording to the video i can not see the notification which trigger with workflow after i click completed.

If i create a notification manually it works fine.

Is there any solution for it ?

**Reply by rskrzypczak** [reply]
Notifications are only created for 1-M relations from the notifications module according to hierarchy.
![image](https://user-images.githubusercontent.com/10221999/109463782-e3f98c00-7a65-11eb-8387-432a6674aaf9.png)

You created an M-M relation.
When you open the notifications list you will see that they were created and related to the right records.
![image](https://user-images.githubusercontent.com/10221999/109463957-2c18ae80-7a66-11eb-99e7-60aa61232b56.png)

These notifications are not displayed in the tabs you showed because this is not the right relation type and unfortunately you won't be able to create it at the user's interface level.
If you have to have the notifications visible in Opportunity preview, even though I frankly don't know why, you will have to correct it directly in the database.
